TodoApp/Tests/TodoApp.Dependency.Tests/Configs/UnityConfigTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using NUnit.Framework;
+using TodoApp.Contracts.Bootstraps;
+using TodoApp.Contracts.Containers;
+using TodoApp.Dependency.Tests.Containers;
+
+namespace TodoApp.Dependency.Tests.Configs
+{
+    [TestFixture]
+    public class UnityConfigTests
+    {
+        private static readonly Type[] ExcludedTypes = 
+        {
+            typeof(IBootstrap),
+            typeof(IDependencyContainer)
+        };
+
+        [Test]
+        public void RegisterDependencies_RegistersRequiredDependencies()
+        {
+            var container = new TestContainer();
+            var contractTypes = typeof(IBootstrap).Assembly.GetTypes();
+            var types = contractTypes
+                .Where(IsNotExcludedInterface)
+                .Append(typeof(HttpRequestMessage));
```

Comment:
I know you have just one right now, however, there might be more and it would seem more natural to me to have another collection (next to `ExcludedTypes`) that could sustain eventually multiple explicit types…